dialect/sql/builder.go

```diff
@@ -2635,6 +2644,9 @@ func (s *Selector) joinUnion(b *Builder) {
 			}
 		}
 	}
+	if s.postgres() {
+		b.WriteString(")")
+	}
 }
```

Comment:
Hey @vzool, please add test cases for this change in `builder_test.go` (for each dialect), so I'll be able to review this change and see the expected query outputs.

Reply:
This PR is only to target `PostgreSQL`, I already updated the `postgres` tests at `TestSelector_UnionOrderBy` and `TestSelector_Union` to check expected query outputs. All tests passed.

Reply:
> 

Done

Reply:
> Hey @vzool, please add test cases for this change in builder_test.go (for each dialect), so I'll be able to review this change and see the expected query outputs.

Done